### DISCUSSION TITLE
MIMIC 3 disease labels

### DISCUSSION THREAD

**Discussion by Khaled-Mohammed-Abdelgaber** [MIT-LCP/mimic-code#1484] [MIMIC-III]
hi everyone. I want to work on disease detection but I could not extract disease labels from mimic 3. I will be thankful for any help.

**Comment by marymlucas** [comment]
Hi @Khaled-Mohammed-Abdelgaber, in the table diagnoses_icd there are ICD9 codes for each admission for a patient, and you can then join this to the d_icd_diagnoses table to get the corresponding disease labels.

Hope this helps.

**Reply by Khaled-Mohammed-Abdelgaber** [reply]
Thanks for your response. I have sent a request to access mimic 3 dataset but the request was refused. Is mimic 2 open to all? If it is open, how can I extract labels?

**Reply by tompollard** [reply]
MIMIC-II is a subset of MIMIC-III. It is no longer available, but it also required users to complete CITI training and provide identity information for review. If you wish to do research using MIMIC, you will need to follow the process for requesting access via PhysioNet. 

If your request was declined and you do not understand why, you should contact the credentialing team to learn more (credentialing@physionet.org). You can then resubmit your request.

**Reply by Khaled-Mohammed-Abdelgaber** [reply]
Thanks for your response. I know that MIMIC-II now is a subset of MIMIC-III. after searching, I get that the MIMIC-III demo is open for everybody. I have downloaded it and I know that it is a subset of MIMIC-III but I can not find the matched records in mimic III. in other words the numeration of the subject id in the PATIENTS.csv file differs from the numeration in the physionet bank atm website.
